packages/settings.gradle.kts

```diff
@@ -19,6 +19,7 @@ include("gradle-plugin")
 include("plugin-compiler")
 include("plugin-compiler-shaded")
 include("library")
+include("sync")
```

Comment:
Maybe `sync-library` and `base-library`? Just to make it clear what they actually are?

Reply:
Maybe `library-base` and `library-sync` just to keep the source trees as neighbours in the IDE?

Reply:
Yeah, that would be better 👍

Reply:
Yeah, that would be better 👍